pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -396,16 +427,19 @@ const (
 	authTypeDefault authType = iota
 	authTypeSharedCreds
 	authTypeKeys
+	authTypeEC2IAMRole
 )
 
 func (at authType) String() string {
 	switch at {
 	case authTypeDefault:
 		return "default"
 	case authTypeSharedCreds:
-		return "sharedCreds"
+		return "credentials"
```

Comment:
What's the rationale behind changing the string representation of this mode?

Reply:
Out of the four auth types, this was the only one that has a string representation that is different from the key that is used to identify the auth type in the provisioning (and in the frontend). Let's say for example that a user has provisioned CloudWatch to use `authType: credentials`. If something goes wrong when attempting to authenticate, grafana will log that `sharedCreds` auth is broken in some way, which might be a bit confusing.

Reply:
OK, makes sense. I don't remember if there was any other usage for the string representation, than just logging it.